src/main/java/cursor/hw12/service/LFUCacheService.java

```diff
@@ -0,0 +1,61 @@
+package cursor.hw12.service;
+
+import cursor.hw12.dao.AverageSalaryDao;
+import cursor.hw12.model.Node;
+import java.util.Map;
+
+public class LFUCacheService {
+    private static final int capacity = 10;
+    private static Map<String, Node> cache;
+    public static LFUCacheService lfuCacheService;
+
+    private LFUCacheService() {
+        cache = AverageSalaryDao.getCache();
+    }
+
+    public static LFUCacheService getInstance() {
+        return lfuCacheService != null
```

Comment:
It's better to use method nonNull() from Objects class 
(Objects.nonNull(lfuCacheService))

Reply:
Resolved